### PR TITLE
tests, utils: coalesce datavolume creation functions

### PR DESCRIPTION
**Pull request by dankenigsberg**
Move DataVolume creation into a single auxiliary function. This makes it clearer to the reader to tell the difference between `NewRandomDataVolumeWithRegistryImportInStorageClass` and `NewRandomBlankDataVolume` (the difference is only the dv source).

This PR drops a needlessly-public constant `TestDataVolumeName` as it becomes an internal implementation detail of a single helper function.

```release-note
NONE
```

### CODE REVIEW

File: tests/utils.go [hunk 1]
```diff
@@ -2071,21 +2066,17 @@ func NewRandomVirtualMachineInstanceWithBlockDisk(imageUrl, namespace string, ac
 	return NewRandomVirtualMachineInstanceWithDisk(imageUrl, namespace, sc, accessMode, k8sv1.PersistentVolumeBlock)
 }
 
-func NewRandomDataVolumeWithRegistryImportInStorageClass(imageUrl, namespace, storageClass string, accessMode k8sv1.PersistentVolumeAccessMode, volumeMode k8sv1.PersistentVolumeMode) *cdiv1.DataVolume {
-	name := TestDataVolumeName + rand.String(12)
-	quantity, err := resource.ParseQuantity("1Gi")
+func newDataVolumeWithManyOptions(namespace, storageClass string, size string, accessMode k8sv1.PersistentVolumeAccessMode, volumeMode k8sv1.PersistentVolumeMode, dataVolumeSource cdiv1.DataVolumeSource) *cdiv1.DataVolume {
```
Comment (brybacki) [inline tests/utils.go:2069]: does it really need the "ManyOptions" suffix?

**Comment by brybacki** [comment]
this is nice, just one question about a name

File: tests/utils.go (hunk 1 above)
Comment (dankenigsberg) [reply tests/utils.go:2069]: naming things is hard. Do you suggest a simple `newDataVolume` ?

Comment (brybacki) [reply tests/utils.go:2069]: yes, I think that would be enough (unless we need to differentiate from another newDataVolume function)

Comment (dankenigsberg) [reply tests/utils.go]: We have other `newDataVolume` entities elsewhere, but not in the `tests` package.

**Comment by brybacki** [comment]
/test pull-kubevirt-e2e-k8s-1.21-sig-storage

Interesting, I have found `{"component":"virt-launcher","level":"error","msg":"No disk capacity","pos":"manager.go:638","timestamp":"2022-01-17T11:02:56.595024Z"}` in virt-launcher logs

**Pull request merged**